perception_dataset/rosbag2/rosbag2_to_non_annotated_t4_converter.py

```diff
@@ -113,7 +115,9 @@ def __init__(self, params: Rosbag2ConverterParams) -> None:
         self._calibrated_sensor_target_frame: str = "base_link"
 
         # Note: To determine if there is any message dropout, including a delay tolerance of 10Hz.
-        self._TIMESTAMP_DIFF = 0.15
+        # changed from 0.15 to 2.0 - a workaround of issue with Unity's step execution
+        # #TODO (mkotynia) should be decreased when the issue with Unity is fixed
+        self._TIMESTAMP_DIFF = 2.0
```

Comment:
I can NOT accept this threshold change in `_TIMESTAMP_DIFF` for all conversion.
Could you avoid this? Why do you need this change?

Reply:
@miursh 
I understand. As described [here](#notes), I needed this change to be able to generate synthetic data because of the issue with too big timestamps mismatch between the camera and lidar corresponding frames which is probably the AWSIM issue. 
Sure, I can avoid it. In commit https://github.com/tier4/tier4_perception_dataset/pull/24/commits/7601e4d41513a1177f2a7586b1d7c37b7695b808 I made timestamp_diff parametrizable through the yaml config file and set the default value of timestamp_diff to 0.15, as it was before my change. Is this solution acceptable for you or do you prefer another one?

Reply:
Thanks. Looks fine